### ISSUE TITLE
Calories Burnt Reported Different in Notification vs In-App

### ISSUE THREAD

**Issue by BattouSaeen** [SecUSo/privacy-friendly-pedometer#175]
Hello, not sure if I'm mistaking units, but the notification that reports kcal, reports one value, e.g. 80 kcal but when you open the app and see the dashboard, it reports it as "0.08 cal". If cal = gram calorie, shouldn't this be 80,000?

I'd suggest this just be the same as the notification (i.e. kcal). Or allow user to select whichever they prefer in settings (ideal) :)

Thanks!

**Comment by FaqatZubair1** [comment]
Exactly, and the units in daily and weekly also differ the same. I'll attach a screenshot for the developer to see what I mean 

![Image](https://github.com/user-attachments/assets/fc54ecf6-30a2-4ef6-a9f7-d3f886a2b9d4)
![Image](https://github.com/user-attachments/assets/6695cb24-8a67-4310-98a5-39b2a441108a)
